pallets/runtime/src/general_tm.rs

```diff
@@ -104,7 +115,7 @@ pub struct RuleData {
     claim: IdentityClaimData,
 
     /// Array of trusted claim issuers
-    trusted_issuers: Vec<IdentityId>,
+    trusted_issuers: Option<Vec<IdentityId>>,
```

Comment:
I think we can use the empty vector case as `None` case.

Reply:
empty vector is sometimes confusing that's why I prefer `None` to take care of that. anyway, we are not allowing an empty vector to be added for the trusted issuer.